sources/views/tabletoolbar.js

```diff
@@ -51,7 +50,7 @@ export default class TableToolbar extends JetView {
 	}
 
 	addItem() {
-		const addedId = this.datatable.add({}, 0);
+		const addedId = this.datatable.add({Name: "Name"}, 0);
```

Comment:
I thought we've already discussed behavior on adding new items and their default values...

Reply:
not necessary to maintain it here, but for cleaner app, we can add empty fields, just do not forget either to add to first pos, ot show item so that users can see the newly added item